guide/src/installation.md

```diff
@@ -1,16 +1,69 @@
 # Install the relayer
 
-In order to install and run Hermes, please follow the steps below:
+There are two main approaches for obtaining Hermes:
 
-> NOTE: This assumes you have installed all the [pre-requisites](./pre_requisites.md) on your machine.
+1. Installation:
+   1. If you are running on a Unix machine (Linux/MacOS), then the simplest
+      option is to [download the latest binary](#Install-by-downloading).
+   2. Alternatively, install via
+      [Cargo installation instructions](#install-via-cargo). 
+
+[comment]: <> (   Regardless of approach you choose, also follow the post-installation)
+
+[comment]: <> (   instructions therein.)
```

Comment:
```suggestion
[comment]: <> (Regardless of approach you choose, also follow the post-installation instructions therein)
```

Reply:
(This was some content I forgot to delete.)